.dev/docker/env/Dockerfile

```diff
@@ -36,44 +34,58 @@ RUN apt-get update \
       libusb-1.0-0-dev \
       libvtk${VTK_VERSION}-dev \
       libvtk${VTK_VERSION}-qt-dev \
+      lsb-release \
       qtbase5-dev \
       software-properties-common \
+      wget \
+      xvfb \
+ && if [ "$USE_LATEST_CMAKE" = true ] ; then \
+      apt-get -V install -y apt-transport-https ; \
+      wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null ; \
+      if [ "$(lsb_release -sr)" = "18.04" ]; then \
+        apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main' ; \
```

Comment:
This can be done using `lsb_release -cs` right?

Reply:
Sadly no, as kitware only provides repositories for the LTS versions of Ubuntu. So it would work with `lsb_release -cs` only for 18.04 and 20.04, but not for 20.10 and 21.04 (where we are using the 20.04 source as well)

Reply:
`lsb_release -d | grep lts -i` ?

Reply:
Instead of relying on hardcoded versions, we can use `if $(lsb_release -d  | grep lts -iq) ; then`

Reply:
ok, found an better way:

```sh
if wget -q --method=HEAD "https://apt.kitware.com/ubuntu/dists/$(lsb_release -cs)/Release"; then \
  apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main" ; \
else \
  apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main' ; \
fi
```